pipeline/container.go

```diff
@@ -39,7 +39,7 @@ type (
 		Number      int               `json:"number,omitempty"      yaml:"number,omitempty"`
 		Ports       []string          `json:"ports,omitempty"       yaml:"ports,omitempty"`
 		Privileged  bool              `json:"privileged,omitempty"  yaml:"privileged,omitempty"`
-		Pull        bool              `json:"pull,omitempty"        yaml:"pull,omitempty"`
+		Pull        string            `json:"pull,omitempty"        yaml:"pull,omitempty"`
```

Comment:
Converting the `Pull` field to a `string` for the `github.com/go-vela/types/pipeline.Container` type